hello_object_oriented/hello_object_oriented.js

```diff
@@ -0,0 +1,320 @@
+// It should have a render method that returns a formatted string to display the task with its name and its assignee.
+// 
+    // myTask.render() // returns 'Clean Dishes'
+// myTaskWithAssignee.render() // returns 'Wash clothes • You'
+
+// I'm constructing a Render Helper to make rendering easier and to avoid 
+// redundant code later.
+
+class RenderHelper{
+    constructor(){
+        this.index = 0;
+    }
+    title(titleString){
+        return `************\n*${titleString}*\n************\n`;
+    }
+    header(headerString){
+        return `|----------\n| ${headerString} \n|----------\n`
+    }
+    listEnumerate(listItem){
+        
+        let curr_string = `| ${this.index}> ${listItem}\n`
+        this.index +=1;
+        return curr_string;
+    }
+    listIndexReset(){
+        //returns an empty line for list formatting
+        this.index = 0;
+        return `| \n`
+    }
+}
+
+
+class Task{
+    constructor(task, name = null){
+        this.task = task;
+        this.name= name;
+    }
+    render() {
+        let retString = this.task;
+        if(!this.name){
+            retString += '.';
+            return retString;
+        } else {
+            retString += ` • `;
+            retString += this.name;
+            retString += ".";
+            return retString;
+        }
+    }
+}
+
+// test
+const newTask = new Task("Wash dishes", "Cam");
+console.log(newTask.render());
+       
+    
+    
+    
+    // Lists
+    // Create a List class that represents a list.It should be initialized with a name.
+// 
+        // List#addTask
+    // It should have a addTask method that takes a Task instance and adds it to the end of the list.You should be able to chain addTask.
+// 
+    // Example Usage:
+// 
+    // const toDoList = new List('To Do');
+// 
+    // adds 'Laundry' task to 'To Do' list
+    // toDoList.addTask(new Task('Laundry', 'You'));
+    // also works by chaining
+    // toDoList
+        // .addTask(new Task('Buy Apples'))
+        // .addTask(new Task('Pay Phone Bill', 'Me'));
+    // List#removeTask
+    // It should have a removeTask method that takes the name of a Task, finds it in the list and removes it.Then, it should return the removed task object or null if it couldn't find a task with that name to remove.
+// 
+    // Example Usage:
+// 
+    // toDoList.addTask('Remove Me!');
+// 
+    // Finds and removes task named 'Remove Me!'
+    // toDoList.removeTask('Remove Me!');
+    // List#render
+    // It should have a render method that returns a formatted string to display the list with its name and all its tasks (using their render methods).
+// 
+        // toDoList.render();
+// 
+    // returns '|---------\n| To Do\n|---------\n| 0> Laundry • You\n| 1> Buy Apples \n| 2> Pay Phone Bill • Me\n|'
+// 
+    // 
+class List{
+
+    constructor(listName){
+        this.listName = listName;
+        this.tasks =[];
+        this.helper = new RenderHelper;
+        }
+    addTask(NewTask){
```

Comment:
Typically you would use lower camel case for arguments (ex `newTask`, and not `NewTask`)